reconcile/gitlab_labeler.py

```diff
@@ -10,6 +11,48 @@
 QONTRACT_INTEGRATION = 'gitlab-labeler'
 
 
+def guess_onboarding_status(changed_paths: List[str]) -> Optional[str]:
+    """
+    Guess the onboarding status of a given MR from the changed paths of the
+    MR.
+    """
+    apps = queries.get_apps()
+    #   logging.info(apps)
+    app_name = get_app_name(changed_paths)
+    if app_name is not None:
+        app = None
+        for a in apps:
+            if a['name'] == app_name:
+                app = a
+
+    if app is not None:
+        logging.info(app['onboardingStatus'])
```

Comment:
Is this a leftover? In any case, you could keep it with a nicer message in debug mode, not info